### DISCUSSION TITLE
Sample CSS for custom CSS creation?

### DISCUSSION THREAD

**Discussion by MajesticalDiscomfort** [baffler/Transparent-Twitch-Chat-Overlay#287] [Q&A]
I know the basics of CSS and want to make a custom format, but I don't know the HTML elements I'm working with and there doesn't seem to be a list or anything on the provided wiki.

I had made one before, but I actually don't remember how I did it, and because I installed the new version, my old CSS isn't there anymore. I just want to try to get it back to the way it was...

I could've sworn that maybe there was a sample CSS in there by default before, but now it only shows a sample for the body class, and not for anything else.

**Comment by MajesticalDiscomfort** [comment]
It looks like someone posted an explanation of their custom CSS which helps with my problem, but I still think the list of relevant classes should be made apparent somewhere.

**Comment by baffler** [comment]
If you click the 3 lines at the top-left and open the menu, you can choose Dev Tools, click "Elements" at the top of that window and you'll be able to see all the HTML elements there. However, I do think it makes a lot of sense to list it out somewhere, I can put it on the wiki 👍

**Reply by MajesticalDiscomfort** [reply]
Oh awesome!! That is really helpful, although the HTML disappears when the message fades, which I could probably adjust in other settings. Although I have noticed that there isn't actually a class for highlighted messages, at least not using KapChat. Do different versions allow the use of highlighted messages?
